src/common/modules/UnicodeFontHandler.js

```diff
@@ -5,146 +5,135 @@ import * as BrowserCommunication from "/common/modules/BrowserCommunication/Brow
 import { isMobile } from "./MobileHelper.js";
 
 import { COMMUNICATION_MESSAGE_TYPE } from "/common/modules/data/BrowserCommunicationTypes.js";
-import { caseIds, fontIds, fonts } from "/common/modules/data/Fonts.js";
+import { CASE, FONT, SEPARATOR, contextMenuList, menuTranslation, fontMap, caseByString, fontByString } from "/common/modules/data/Fonts.js";
```

Comment:
Note that this will include four duplicate copies of the strings, which will both increase memory use and make this feature more difficult to maintain.

Reply:
Yeah, it exaggerated a little yesterday as I "only" wanted to use Symbols as IDs, but then stumbled upon all these errors that made it harder than it should be. E.g. you can only use strings in the `contextMenu` API as an ID – respectively if you use a Symbol it does not match the Symbol that was passed to it, before.
Still need to think whether there may be a better solution.